lib/android/app/src/main/java/com/reactnativenavigation/views/StackLayout.java

```diff
@@ -34,6 +41,10 @@ public void onPress(String buttonId) {
 
     public void applyOptions(Options options, ReactComponent component) {
         new OptionsPresenter(topBar, component).applyOptions(options);
+        if (fabOptionsPresenter == null) {
```

Comment:
Our code should be concise and deterministic. if `fabOptionsPresenter` is a property of the class, it should not be initialized conditionally on demand, instead create in ,like all fields, in constructor and inject the `component` in `applyOptions` method.